include/ringmesh/geomodel/builder/geomodel_builder_remove.h

```diff
@@ -290,7 +290,7 @@ namespace RINGMesh
         friend class GeoModelBuilderBase< DIMENSION >;
         friend class GeoModelBuilder< DIMENSION >;
 
-    private:
+    public:
```

Comment:
Why this change? Was this builder accessible before?

Reply:
I did not manage to instantiate this builder since its constructor was private